### DISCUSSION TITLE
Clicking scenario title is now showing loot table & monsters in unrevealed room. How can I turn that off?

### DISCUSSION THREAD

**Discussion by Solmyr14** [Lurkars/gloomhavensecretariat#353] [Q&A] [answered]
I've jumped from v0.69.1 to v0.71.4 and now when I click on either the scenario title or the next room to unlock/reveal the window that pops up shows the loot table and all the enemies for the whole scenario (including unrevealed rooms) spoiling both things.

How can I turn that off so that clicking the next room will only bring up the confirmation box and clicking the scenario title doesn't show spoilers?

Cheers.

**Comment by Lurkars** [comment, marked as answer]
I am confused, first of all this only shows (should!) data visible on the initial page of the scenario book. So I won't call this spoiler. There is extra functionality with a `Spoiler` button on certain scenarios when there are changes to monsters not visible on the initial page. So not spoiling is an extra use case considered. Second, this should only pop-up before first round. So after first draw, the setup is hidden on that menu (and can only be shown with a toggle). So if you don't want to see the things from the scenario book, just don't click the menu before first drawn. This is all in purpose, because the menu without setup is useless before first draw, because you also won't be able to open rooms, treasures etc. without playing. So you can just ignore it before first round. But as said, it should not spoiler anything. If you really find a spoiler, something shown not in the initial scenario page, then please tell me, because that should not happen (without clicking `Spoiler`).

**Comment by Solmyr14** [comment]
Sorry, I should have be more clear, spoiler for my group. We don't use the scenario book. We use Secretariat and Frosthaven Scenario Viewer so we only see what is revealed (no loot table, enemies, number of unrevealed tiles, etc...). I only called it a spoiler since in the settings the info "i" beside "Add for all rooms/scenarios" says the last sentence says "Please consider that this may spoiler unrevealed rooms/sections", so I though you were considering anything beyond the first room a spoiler. No biggie.

I found this out this evening while getting ready for tomorrow's game session and noticed you made a number of updates. I was checking it out when I ran across the new Scenario Setup window, but did not actually go beyond opening a scenario as we aren't playing yet, so no round was started. Now that you have pointed out it goes away once the first round starts, it is no longer a problem.

Thank you again for your work and helping me these few times lately.

Cheers!
